storage/innobase/handler/ha_innodb.cc

```diff
@@ -964,10 +976,18 @@ static SHOW_VAR innodb_status_variables[]= {
   {"buffer_pool_pages_latched",
   (char*) &export_vars.innodb_buffer_pool_pages_latched,  SHOW_LONG},
 #endif /* UNIV_DEBUG */
+  {"buffer_pool_pages_made_not_young",
+  (char*) &export_vars.innodb_buffer_pool_pages_made_not_young, SHOW_LONG},
+  {"buffer_pool_pages_made_young",
+  (char*) &export_vars.innodb_buffer_pool_pages_made_young, SHOW_LONG},
   {"buffer_pool_pages_misc",
   (char*) &export_vars.innodb_buffer_pool_pages_misc,	  SHOW_LONG},
+  {"buffer_pool_pages_old",
+  (char*) &export_vars.innodb_buffer_pool_pages_old,	  SHOW_LONG},
   {"buffer_pool_pages_total",
   (char*) &export_vars.innodb_buffer_pool_pages_total,	  SHOW_LONG},
+  {"buffer_pool_pages_LRU_flushed",
+  (char*) &export_vars.innodb_buffer_pool_pages_LRU_flushed, SHOW_LONG},
```

Comment:
Please avoid introducing members to `export_vars` except when it is really necessary. I recently added the following:
```c
  {"available_undo_logs", &srv_available_undo_logs, SHOW_ULONG},
```
No type cast, and using `SHOW_ULONG` instead of the misleading alias name `SHOW_LONG`.

For this to work, the variable has to be declared `ulong` (or `ulonglong`) instead of the traditional InnoDB integer data type `ulint`.

Reply:
Similarly, should I expose `srv_log_writes_and_flush`, `srv_main_active_loops` and `srv_main_idle_loops` so we can directly reference them in `innodb_status_variables`, bypassing `export_vars`?

Reply:
Yes, please do that.